docs/en/observability/apm/upgrading-to-integration.asciidoc

```diff
@@ -174,6 +174,8 @@ The process of migrating should only take a few minutes.
 
 With a Superuser account, complete the following steps:
 
+//QUESTION: Not sure how to handle the following instruction since the Settings are hidden under the app and can't be searched for. A user has to select one of the pages under Application, but not all of the pages have the Settings link. It's weird. Do you think I should change this or just leave it as-is?
```

Comment:
This is related to the apm vs applications problem I noted earlier. You need to search on APM in order to end up in one of the pages in the Applications app.

Reply:
I'm going to side-step the problem by saying: "In {kib}, go to the **Applications** app and click **Settings** → **Schema**"